packages/devtools_app/lib/src/screens/network/network_controller.dart

```diff
@@ -380,13 +380,13 @@ class NetworkController
   @override
   void filterData(Filter<NetworkRequest>? filter) {
     serviceManager.errorBadgeManager.clearErrors(NetworkScreen.id);
-    if (filter?.queryFilter == null) {
+    if (filter == null || filter.queryFilter == null) {
```

Comment:
maybe my comment was a little misleading. I meant this:
```
final queryFilter = filter?.queryFilter;
if (queryFilter == null) {
  ...
} else {
  // no need to create the var and null assert here
  filteredData...
}
```

Reply:
Ah, good call. That works better.